csrankings-r.csv

```diff
@@ -391,6 +391,7 @@ Ravi Janardan,University of Minnesota,http://umn.edu/home/janardan,NOSCHOLARPAGE
 Ravi K. Iyer,Univ. of Illinois at Urbana-Champaign,https://www.ece.illinois.edu/directory/profile/rkiyer,NOSCHOLARPAGE
 Ravi Karkar,University of Massachusetts Amherst,https://homes.cs.washington.edu/~rkarkar,n_vk_EAAAAAJ
 Ravi Mukkamala,Old Dominion University,http://www.cs.odu.edu/~mukka,NOSCHOLARPAGE
+Ravi Netravali,Princeton University,https://www.cs.princeton.edu/~ravian,4C9naMgAAAAJ

```

Comment:
In industry (at Reulay).

Reply:
Netravali is at Princeton, advising students, and teaching for the Fall:

https://www.cs.princeton.edu/~ravian/
https://www.cs.princeton.edu/courses/archive/fall22/cos316/

Reply:
Needs to be removed unless they are back full-time, i.e., 75%+ over the academic year.

Reply:
I'm not in a position to have that info, so pulled for now until I can get better information. Thanks.